src/status_im/navigation/core.cljs

```diff
@@ -390,7 +390,9 @@
                          :create-community-channel
                          :community-emoji-thumbnail-picker
                          :create-community-category
-                         :community-edit-chats))
+                         :community-edit-chats
+                         :community-edit
```

Comment:
how is this change related to this PR ?

Reply:
Hi @flexsurfer, Thanks for reviewing PR
Please check this comment.
https://github.com/status-im/status-react/pull/13157#discussion_r821511220